src/content_script.js

```diff
@@ -26,9 +26,15 @@ const SHARE_PATTERN_DETECTION_SELECTORS = [
   "[href*='facebook.com/share']", // Imgur Login
   "[href*='facebook.com/dialog/share']",
   "[href*='facebook.com/sharer']", // Buzzfeed
-  "[aria-label*='share on facebook']" // MSN
+  "[data-bfa-network*='facebook']", // Buzzfeed Mini Share
+  "[aria-label*='share on facebook']", // MSN
+  "[title='Share on Facebook']" // Medium
 ];
 
+// const SHARE_DISABLE_DETECTION_SELECTORS = [
+//   "[class*='js-share-button--facebook']" // Buzzfeed
+// ];
```

Comment:
Need to keep this? Or can we delete it completely?

Reply:
Migrated to a TO-DO.